src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/entity/CertificateTypeDB.kt

```diff
@@ -19,14 +19,12 @@
 
 package org.eclipse.tractusx.bpdmcertificatemanagement.entity
 
-import jakarta.persistence.Column
-import jakarta.persistence.Entity
-import jakarta.persistence.Table
+import jakarta.persistence.*
 
 @Entity
 @Table(name = "certificate_types")
 class CertificateTypeDB(
-    @Column(name = "certificate_type", nullable = false)
+    @Column(name = "certificate_type", unique = true, nullable = false)
```

Comment:
You sure this is unique? If both certificate version and certificate type are needed to identify the actual certificate type I would assume the same certificate type will be in the database with different versions. So I guess certificate type key and version pair are unique but not the individual fields.

Reply:
Yeah, I had call with people involved on requirement clarification. 
And made changes to make pair i.e. certificate type and certificate version as unique.